code/components/citizen-resources-gta/src/GameEventFunctions.cpp

```diff
@@ -9,6 +9,41 @@
 
 static InitFunction initFunction([]
 {
+	OnTriggerGameEventReact.Connect([](const GameEventData& data)
+	{
+		auto resman = Instance<fx::ResourceManager>::Get();
+		auto rec = resman->GetComponent<fx::ResourceEventManagerComponent>();
+
+		/*NETEV gameEventReact:CEventName CLIENT
+		/#*
+		 * An event that is triggered when the game triggers an internal network event.
```

Comment:
It may be helpful to add here that 'CEventName' is just a placeholder.

Reply:
I squashed all changes after the first review, gave it a feat(*) styled title, and added explanation for the CEventName.

Let me know if this is to your liking.